### PR TITLE
Introduce sound backend

### PR DESCRIPTION
**Pull request by iartemenko**
Add sndbe_git.bb recipe file.

### CODE REVIEW

File: machine/meta-rcar-gen3/recipes-extended/sndbe/sndbe_git.bb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+DESCRIPTION = "sndbe"
```
Comment (andr2000) [inline machine/meta-rcar-gen3/recipes-extended/sndbe/sndbe_git.bb]: Para-virtualized sound backend

**Review by andr2000 — CHANGES_REQUESTED** [review]

**Comment by andr2000** [comment]
commit message: backend, not background

**Comment by andr2000** [comment]
-1

**Comment by iartemenko** [comment]
Done

**Comment by andr2000** [comment]
lgtm, rb

**Pull request merged**